### PR TITLE
Add homebrew services integration

### PR DESCRIPTION
**Pull request by el7cosmos**
Utilize homebrew services, so it can be used like:
```shell
brew services <start|restart|stop> unit
```

### CODE REVIEW

File: Formula/unit.rb [hunk 1]
```diff
@@ -32,6 +32,12 @@ def post_install
     (var/"run/unit").mkpath
     (var/"state/unit/certs").mkpath
   end
+  
+  plist_options manual: "unitd --no-daemon"
```
Comment (mar0x) [inline Formula/unit.rb]: What is the purpose of this line? It seems brew services works fine without it.

**Comment by mar0x** [comment]
Thanks for contribution!
Please update the commit message title to something like _"Added homebrew services integration."_ to conform Unit commit message rules.

File: Formula/unit.rb (hunk 1 above)
Comment (el7cosmos) [reply Formula/unit.rb]: I tried, and I can't find any effect of that. Just some core Formulas having that line before service block. Will omit that.

**Comment by mar0x** [comment]
Please remove trailing whitespace at the line 35 to make test-bot happy.

**Pull request merged**